data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -20,6 +20,18 @@
             </column>
         </createTable>
 
+        <createTable tableName="utxo_transaction_status">
```

Comment:
@mnesbit @blsemo 
I can't remember what we agreed in the end on this vs a field in on the tx table, if the purpose is an audit trail of status changes, then I think we should consider either:
- adding a version # to the tx and the status table to make the join 1-1
- adding the current status to the tx table and keep the status table as a pure audit table.

Reply:
We agreed to left it out until we are clear what we need, but Dan said he needs it now in some form so I've added a table that is same as we have for consensual ledger, and we can modify it later.